### ISSUE TITLE
 Left/right arrow buttons in activity have not accessible name

### ISSUE THREAD

**Issue by akhanfir** [Meeds-io/meeds#1304]
**Steps to reproduce**

1. Create a space

2. Open activity composer drawer

3. Create an activity with 3 or more attachments

4. Launch a lighthouse report related to accessibility

**Current behaviour**

Left/right arrow buttons in published activity are reported as not accessible..

**Expected behavior**

Left/right arrow buttons must be accessible, you must add the aria-label attribute:

- For the left button "Previous visual"

- For the right button “Next visual”

Both labels should be i18n.
